### ISSUE TITLE
Missign writeVTK switch in actuatorLineSource::addSup in branch al-viz

### ISSUE THREAD

**Issue by chegdan** [turbinesFoam/turbinesFoam#222]
Within the function definition 

```
void Foam::fv::actuatorLineSource::addSup
(
    const volScalarField& rho,
    fvMatrix<vector>& eqn,
    const label fieldI
)
```
 the call to write the VTK needs to be adjusted since it is missing a flag.  The call

```
    // write the VTK file
    if (mesh_.time().outputTime() && Pstream::master())
    {
        writeVTK();
    }
```
should be changed to 
```
    // Write the VTK file
    if
    (
        writeVTK_ &&
        mesh_.time().outputTime() &&
        Pstream::master()
    )
    {
        writeVTK();
    }
```

**Comment by petebachant** [comment]
This was fixed by #223, right Dan? Okay to close?

**Comment by chegdan** [comment]
yep, can be closed.
